discovery-provider/src/challenges/profile_challenge.py

```diff
@@ -0,0 +1,116 @@
+from collections import Counter
+from src.models import ProfileCompletionChallenge, Challenge, User, AggregateUser, Repost, UserChallenge, Follow, Save
+from src.challenges.challenge_event_bus import ChallengeEvent
+from src.challenges.challenge import ChallengeManager, ChallengeUpdater
+
+REPOST_THRESHOLD = 1
+FOLLOW_THRESHOLD = 5
+FAVORITES_THRESHOLD = 1
```

Comment:
we should totes just expose an endpoint
/v1/users/profile_complete/<id>
to handle the client logic for the completion meter
(not now obv)

Reply:
100%